blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/AbstractPipelineImpl.java

```diff
@@ -231,6 +233,22 @@ public Link getLink() {
         };
     }
 
+    @Override
+    public List<Object> getParameters() {
+        return getParameterDefinitions(job);
+    }
+
+    public static List<Object> getParameterDefinitions(Job job){
```

Comment:
Ah thanks, I think I know this plugin, thought this work is in separate plugin:) 

I hope branches are sorted (by name) and not given in the order of declaration. I think I see it working for simpler cases, for others (like groovy script creating branches etc.) it might not. Or maybe PMD doesn't allow arbitrary groovy script in them.